items.json

```diff
@@ -109789,21 +109789,21 @@
         "thrustDamage": 0,
         "thrustDamageType": "Undefined",
         "thrustSpeed": 90,
-        "swingDamage": 28,
+        "swingDamage": 27,
         "swingDamageType": "Blunt",
         "swingSpeed": 86
       }
     ]
   },
   {
-    "id": "crpg_militia_pernach_v1_h0",
-    "baseId": "crpg_militia_pernach_v1",
+    "id": "crpg_militia_pernach_v2_h0",
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.